sunkit_pyvista/tests/test_plotting.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+import pfsspy
+import pytest
+import pyvista
+from pfsspy import tracing
+from pfsspy.sample_data import get_gong_map
+
+import astropy.constants as const
+import astropy.units as u
+import sunpy.data.test as test
+import sunpy.map as smap
+from astropy.coordinates import SkyCoord
+from sunpy.coordinates import frames
+
+from sunkit_pyvista import SunpyPlotter
+
+pyvista.OFF_SCREEN = True
+
+
+@pytest.fixture
+def aia171_test_map():
+    return smap.Map(test.get_test_filepath('aia_171_level1.fits'))
+
+
+@pytest.fixture
+def plotter():
+    return SunpyPlotter()
+
+
+def test_plot_map_with_functionality(aia171_test_map, plotter, verify_cache_image):
```

Comment:
Might be better to increase the width of the rectangle? I can barely see it on the main image.

Reply:
This has to be changed based on input from #49.
I shall update the figures when that changes as altering `line_width` does result in erros